geonode/assets/local.py

```diff
@@ -62,6 +62,18 @@ def create(self, title, description, type, owner, files=None, clone_files=False,
 
         if clone_files:
             files = self._copy_data(files)
+        elif isinstance(files[0], str):
```

Comment:
What if the second file in the list is a "InMemoryFile"? I think here is better to loop on the list rather then accessing to the index 0